app/notifications/rest.py

```diff
@@ -11,8 +11,8 @@
 )
 
```

Comment:
Changes in this file are about using the new response objects defined in the clients.